mono/mini/mini-runtime.h

```diff
@@ -510,11 +510,23 @@ void mono_enable_jit_map (void);
 void mono_emit_jit_map   (MonoJitInfo *jinfo);
 void mono_emit_jit_tramp (void *start, int size, const char *desc);
 gboolean mono_jit_map_is_enabled (void);
+#if defined(ENABLE_JIT_DUMP)
+#include <sys/mman.h>
+#include <sys/syscall.h>
+#include "/usr/include/elf.h"
+void mono_enable_jit_dump (void);
+void mono_emit_jit_dump (MonoJitInfo *jinfo, gpointer code);
+void mono_jit_dump_cleanup (void);
+#endif
 #else
 #define mono_enable_jit_map()
 #define mono_emit_jit_map(ji)
 #define mono_emit_jit_tramp(s,z,d) do { } while (0) /* non-empty to avoid warning */
 #define mono_jit_map_is_enabled() (0)
+
+#define mono_enable_jit_dump()
```

Comment:
Instead of macros, consider making the functions empty, or use runtime `if` in the functions and avoiding the preprocessor entirely. Is it even much code to omit? `#if` is more trouble than people realize -- having to reconfigure to turn things on/off.